### PR TITLE
[JFG-944] java builders - new smoke test

### PR DESCRIPTION
**Pull request by oskliarov**

### CODE REVIEW

File: test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTestSuite.java [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.IterationsNumber;
+import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.MaxDurationInSeconds;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+
+@Configuration
+public class JaggerSmokeTestSuite extends JaggerPropertiesProvider {
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/java/com/griddynamics/jagger/test/javabuilders/JaggerSmokeTestSuite.java]: test suite => load scenario

File: test/java-builder-tests/src/main/resources/profiles/basic/environment.properties [hunk 2]
```diff
@@ -16,6 +16,8 @@
 #
 
 
+test.service.endpoints=http://localhost:8080
```
Comment (dlatnikov) [inline test/java-builder-tests/src/main/resources/profiles/basic/environment.properties]: propose to move to test.properties

Comment (oskliarov) [inline test/java-builder-tests/src/main/resources/profiles/basic/environment.properties]: this property depends on environment, i think it's right place.

Comment (dlatnikov) [inline test/java-builder-tests/src/main/resources/profiles/basic/environment.properties]: we agreed to leave env.properties for launching framework (in stand by mode). All parameters, related to the test execution are the part of test properties now. This particular case is a good chance to verify our new approach ;)

**Pull request merged**